golangci-lint | declaration has 3 blank identifiers (dogsled)

```log
##[error]logging/logging.go:197:2: declaration has 3 blank identifiers (dogsled)
	_, _, line, _ := runtime.Caller(1)
	^
##[error]config/config_test.go:34:2: declaration has 3 blank identifiers (dogsled)
	_, filename, _, _ := runtime.Caller(1)
	^
```

Exposed when moving the linting settings from shell script to external config file as part of #236. Haven't double-checked yet to see if I've unintentionally enabled additional linters when copy/pasting the file from the atc0005/send2teams project.

--- a/config/config_test.go
+++ b/config/config_test.go
@@ -31,7 +31,8 @@ import (
 func GetBaseProjectDir(t *testing.T) string {
 
 	// https://stackoverflow.com/questions/23847003/golang-tests-and-working-directory
-	_, filename, _, _ := runtime.Caller(1)
+	// TODO: How else to retrieve only the one value that I need? See GH-237.
+	_, filename, _, _ := runtime.Caller(1) // nolint:dogsled
 	// The ".." reflects the path above the current working directory
 	dir := path.Join(path.Dir(filename), "..")
 	return dir

--- a/logging/logging.go
+++ b/logging/logging.go
@@ -194,6 +194,7 @@ func SetLoggerLevel(logger *logrus.Logger, logLevel string) error {
 // line number from the point this function was called.
 // TODO: Find a better location for this utility function
 func GetLineNumber() int {
-	_, _, line, _ := runtime.Caller(1)
+	// TODO: How else to retrieve only the one value that I need? See GH-237.
+	_, _, line, _ := runtime.Caller(1) // nolint:dogsled
 	return line
 }

